docs/configuration/module/traceroute.mdx

```diff
@@ -41,7 +41,7 @@ Under the node list, long hold a destination node and select 'Traceroute' to sen
 
 Make sure the app is at least version 2.0.9.
 
-Under Contacts > Direct Messages, long hold a destination node and select 'Trace Route' to send the request. Depending on the amount of hops that is needed, this might take a while. The result will be shown in the Mesh Log.
+Under the node list, long hold a destination node and select 'Trace Route' to send the request. Depending on the amount of hops that is needed, this might take a while. The result will be shown in the Mesh Log.
```

Comment:
Sure - do you want the same change for Android, too? cf lines 31-44:

`#### Android

Make sure the app is at least version 2.1.10.
  
Under the node list, long hold a destination node and select 'Traceroute' to send the request. Depending on the amount of hops that is needed, this might take a while. The result will be shown using a pop-up.

</TabItem>
<TabItem value="apple">

#### Apple

Make sure the app is at least version 2.0.9.

Under the node list, long hold a destination node and select 'Trace Route' to send the request. Depending on the amount of hops that is needed, this might take a while. The result will be shown in the Mesh Log.
`

Reply:
Have updated Apple only for now; and also updated where the result will be shown.

Reply:
Done! ...I think you can accept that now or do I need a new PR?

Reply:
Unfortunately hitting the button at the bottom of  https://cla-assistant.io/meshtastic/ I get a 404 error so can't sign.  - Who manages this that can help sort this out?